src/controllers/auth.js

```diff
@@ -31,7 +31,10 @@ const login = async (req, res) => {
   const tokens = await authService.login(email, password, { rememberMe })
 
   const refreshTokenCookieOptions = {
-    ...COOKIE_OPTIONS,
+    httpOnly: true,
+    secure: true,
+    sameSite: 'none',
+    domain: COOKIE_DOMAIN,
     maxAge: rememberMe ? thirtyDaysInMs : oneDayInMs
```

Comment:
```suggestion
    ...COOKIE_OPTIONS,
    maxAge: rememberMe ? thirtyDaysInMs : oneDayInMs
```
Here you repeat data from `COOKIE_OPTIONS` with difference in `maxAge` field 
So i think you can leave `...COOKIE_OPTIONS` here and only add new `maxAge` to update it